freecad/Curves/Birail.py

```diff
@@ -1,3 +1,10 @@
+# -*- coding: utf-8 -*-
+
+__title__   = "Birail"
```

Comment:
This is not compliant with PEP8 : [https://www.flake8rules.com/rules/E221.html](https://www.flake8rules.com/rules/E221.html)